app/src/main/java/com/example/tasky/agenda_details/presentation/AgendaDetailsViewState.kt

```diff
@@ -13,10 +14,11 @@ data class AgendaDetailsViewState(
     val fromTime: String = "",
     val toDate: String = "",
     val fromDate: String = "",
-    val reminder: Int = 0,
+    val reminderTime: ReminderTime = ReminderTime.THIRTY_MINUTES,
+    val showReminderDropdown: Boolean = false,
     val attendeeFilterSelected: AttendeeFilter = AttendeeFilter.ALL
 )
 
-enum class AttendeeFilter(type: String) {
-    ALL("All"), GOING("Going"), NOT_GOING("Not Going")
+enum class AttendeeFilter(val typeName: String) {
+    ALL("All"), GOING("Going"), NOT_GOING("Not going")
 }
```

Comment:
If you work with string resources all over the place, could it make sense to also do that here? 😄